src/en/hentairead/src/eu/kanade/tachiyomi/extension/en/hentairead/Hentairead.kt

```diff
@@ -17,6 +19,8 @@ import java.util.Locale
 
 class Hentairead : Madara("HentaiRead", "https://hentairead.com", "en", dateFormat = SimpleDateFormat("dd/MM/yyyy", Locale.US)) {
 
+    override val json = Json { ignoreUnknownKeys = true }
```

Comment:
what's wrong with the singelton json instance?

Reply:
I thought it was erroring out, but when retesting that's not the case. I'll remove it. Probably ran into the issue of fixing two things at once 🤐